requirements.txt

```diff
@@ -0,0 +1,8 @@
+flask>=0.10.0
+lxml>=3.4.4
+spyne>=2.12.8
+python-dateutil
+jinja2
+pytz
+sqlalchemy
+odm2api
```

Comment:
That makes sense! I didn't bother to look for references to it, sorry. I didn't think very hard about "dev"; now I realize that it's WOFpy + the test suite.